elevenclock/lang/lang_he.py

```diff
@@ -0,0 +1,119 @@
+# INSTRUCTIONS
+
+# Translate the text and write it between the "
+# EXAMPLE:      original    ->  "This text is in english: value {0}" 
+#               translation ->  "Aquest text està en anglès: valor {0}"
+
+# So it would look like: "ORIGINAL_TEXT" : "TRANSLATED_TEXT",
+
+
+# If you see sth like {0}, {1}, maintain it on the translated sentence
+# Meke special attention to elements like ":", etc.
+
+lang_2_8 = {
+    "Force the clock to be at the top of the screen": "קבע תצוגת שעון בחלק העליון של המסך",
+    "Show the clock on the primary screen": "הצג שעון במסך הראשי",
+    "Use a custom font color": "קביעת צבע גופן",
+    "Use a custom background color": "קביעת צבע לרקע השעון",
+    "Align the clock text to the center": "מרכז את טקסט השעון",
+    "Select custom color": "בחר צבע",
+    "Hide the clock when a program occupies all screens": "הסתר את השעון כאשר תוכנה תופסת את כל המסכים",
+}
+
+lang2_7_bis = lang_2_8 | {
+    "Use a custom font": "בחר סוג גופן",
+    "Use a custom font size": "בחר גודל גופן",
+    "Enable hide when multi-monitor fullscreen apps are running": "הפעל הסתרת שעון, כאשר אפליקציות מסך מלא בריבוי מסכים פועלת",
+    "<b>{0}</b> needs to be enabled to change this setting": "חייב להיות פעיל, על מנת לשנות הגדרה זאת",
+    "<b>{0}</b> needs to be disabled to change this setting": "חייב להיות כבוי, על מנת לשנות הגדרה זאת",
```

Comment:
You are missing the <b>{0}</b> Those code for an inner string that willl be dinamicallly added